inputs/sortIndex_04.json

```diff
@@ -1 +1,7 @@
-{"sortIndex.downsample_docker":"${workspace.docker}","sortIndex.reference_fasta":"${workspace.reference_fasta}","sortIndex.sort_disk_size":"${100}","sortIndex.sort_mem_size":"3 GB","sortIndex.cram_downsample_file":"${this.downsample_cram}"}
\ No newline at end of file
+{
```

Comment:
It would probably be good to clean up the JSONs that you don't need, and make sure that they're up to date with the new style of runtime attributes.